accessibility-checker-engine/src/v4/rules/meta_refresh_delay.ts

```diff
@@ -43,18 +44,39 @@ export const meta_refresh_delay: Rule = {
         "level": eRulePolicy.VIOLATION,
         "toolkitLevel": eToolkitLevel.LEVEL_THREE
     }],
-    act: [ "bisz58", "bc659a" ],
+    //act: [ "bisz58", "bc659a" ],
+    act: [ "bc659a" ],  // pass if a page is redirected after more than 20 hours (7200)
     run: (context: RuleContext, options?: {}, contextHierarchies?: RuleContextHierarchy): RuleResult | RuleResult[] => {
         const ruleContext = context["dom"].node as Element;
         if (ruleContext.getAttribute("http-equiv").toLowerCase() !== 'refresh')
             return null;
 
+        let doc = ruleContext.ownerDocument;
+        if (!doc) return;
+
+        // check if the rule already passed: the first one takes priority
+        if (CacheUtil.getCache(doc, "meta_refresh_delay_done", false))
+            return null;
+
         let content = ruleContext.getAttribute("content").toLowerCase();
+        if (!content || content.trim().length ===0)
+            return null;
+
+        let time:number = -1;
+        if (content.match(/^\d+$/)) 
+            time = parseInt(content);
+        else if (content.match(/^\d+;/)) {
+            let pos = content.indexOf(";");
+            time = parseInt(content.substring(0, pos));
+        }
         // Invalid content field
-        if (!content.match(/^\d+$/) && !content.match(/^\d+;/)) {
+        if (time === -1) {
```

Comment:
Same as above, should this be time < 0?

Reply:
the time was set up in the line 65: let time:number = -1;  so if no match, then the time was not changed, or I misunderstood?